### ISSUE TITLE
Upgrade to Nightwatch 3

### ISSUE THREAD

**Issue by lostcarpark** [ddev/ddev-selenium-standalone-chrome#54]
I'm running Nightwatch tests with ddev-drupal-contrib, which is currently it's running Nightwatch 2.4.2.

However, GitlabCi is now running Nightwatch 2.4.2 for Drupal <= 11.0 and 3.9 for Drupal >=11.1.

Is it possible to have Nightwatch 3.9 install for ddev-drupal-contrib?

Ideally it would be great to have 2.4 and 3.9 installed simultaneously, but not sure if that's feasible in the same DDEV instance.

**Comment by weitzman** [comment]
This plugin doesnt actually specify nightwatch at all. That must be coming from elsewhere, possibly drupal core

**Comment by rfay** [comment]
You're right, I incorrectly moved this from ddev-drupal-contrib for some reason. The actual nightwatch command is there, https://github.com/ddev/ddev-drupal-contrib/blob/main/commands/web/nightwatch and it obviously runs nightwatch via yarn, which means this is what's installed via composer. I'll move this back. Hope that makes sense @lostcarpark - it's all about your project composer setup.
